detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactNativeLoadingMonitor.kt

```diff
@@ -69,8 +64,10 @@ open class ReactNativeLoadingMonitor(
                 // Due to an ugly timing issue in RN
                 // it is possible that our listener won't be ever called
                 // That's why we have to check the reactContext regularly.
```

Comment:
Can we Detox this comment a bit? 😄 Is it really an _ugly_ unexplained timing issue or do we understand things better now and give a good technical explanation (for future generations' sake)